### PR TITLE
Chore: Update proxyquire

### PR DESCRIPTION
**Pull request by sarvaje**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [x] Added/Updated related documentation.
- [x] Added/Updated related tests.

### CODE REVIEW

**Comment by molant** [comment]
@sarvaje it looks like there's an issue with `extension-browser` in this PR that fails on all platforms.

Can you please take a look?

**Comment by sarvaje** [comment]
@molant it should be fixed now

File: packages/extension-browser/tests/background-script.ts [hunk 1]
```diff
@@ -50,7 +49,7 @@ const prepareContentScriptInjection = (sandbox: SinonSandbox, browser: typeof ch
     };
 };
 
-test('It registers for messages.', async (t) => {
+test.serial('It registers for messages.', async (t) => {
```
Comment (molant) [inline packages/extension-browser/tests/background-script.ts]: 😢

Comment (antross) [reply packages/extension-browser/tests/background-script.ts]: This shouldn't be necessary. Something must be missing we need to reset when stubbing globals...

Comment (sarvaje) [reply packages/extension-browser/tests/background-script.ts]: Sorry, this wasn't necessary, it was something that I tested and forgot to restore.

**Pull request merged**